### PR TITLE
M sc1 59 19

### PR DESCRIPTION
**Pull request by Yarrik**

### CODE REVIEW

**Comment by Yarrik** [comment]
Please check this tasks

**Comment by Yarrik** [comment]
Please check this tasks

File: OYarotski/02.Tables,links/1.Calendar.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <HEAD> 
+      <META charset="utf-8"> 
+      <TITLE>Таблицы</TITLE> 
+    </HEAD> 
+    <BODY> 
+        <table border=1 width="400" align="center">
+        <caption>Месяц моего дня рождения</caption>
+        <col width="40"><col width="30"><col width="30"><col width="30"><col width="30"><col width="30"><col width="30"> 
+        <tr><th colspan="6">Май</th><tr></tr>
+        <tr><th>ПН</td><td></td><td>6</td><td>13</td><td>20</td><td>27</td><tr>
```
Comment (pluhin) [inline OYarotski/02.Tables,links/1.Calendar.html]: the is open tag tr in the end of the lines 12-18. should be close part, not open part

File: OYarotski/02.Tables,links/2.Weather.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html> 
+<HTML lang="ru"> 
+    <HEAD> 
+      <META charset="utf-8"> 
+      <TITLE>Таблицы</TITLE> 
+    </HEAD> 
+    <BODY> 
+        <table border=1 width="400" align="center">
+        <caption><i>Прогноз погоды</i></caption>
+        <tr><th>Город</th><th>Утро</th><th>День</th><th>Вечер</th><th>Ночь</th><tr>
```
Comment (pluhin) [inline OYarotski/02.Tables,links/2.Weather.html]: the is open tag tr in the end of this line. should be close part, not open part

File: OYarotski/02.Tables,links/Python_1.html [hunk 3]
```diff
@@ -4,51 +4,60 @@
...
         программирование графических интерфейсов <ins>(GUI)</ins>. Давайте создадим простую программу прямо
         сейчас.</p>
-        <h2>Наша первая программа</h2>
+        <h2 id="first programm">Наша первая программа</h2>
+        <p><a href="#content">К содержанию</a></p>
         <p>Наберём код в текстовом редакторе (например, в gedit). Потом мы разберёмся, что делает каждая
         строка и пойдём дальше.</p>
         <p>Введите эти 4 строки:</p>
-<pre>#!/usr/bin/env python
+<pre>#!/usr/bin/env pythonU CYU.6KD..M,
```
Comment (pluhin) [inline OYarotski/02.Tables,links/Python_1.html]: What is going with python interpreter?

**Review by pluhin — CHANGES_REQUESTED** [review]
All is fine, except tables, see comments

**Review by pluhin — APPROVED** [review]
All ok, approved

**Pull request merged**